packages/foreman/foreman/foreman.spec

```diff
@@ -6,10 +6,11 @@
 
 # explicitly define, as we build on top of an scl, not inside with scl_package
 %{?scl:%global scl_prefix %{scl}-}
+%{?scl:%global _scl_root /opt/theforeman/%{scl}/root}
```

Comment:
I added this to centralize this reference. The trick with any RPM built against but not into the SCL is that most of the SCL macros are not defined and therefore have to be defined manually.

Reply:
This should be part of the RPM macros. I wonder why it's not set properly. Should it be provided by tfm-build?

Reply:
It could be useful to create special macro to represent this path.

Reply:
Perhaps we should have a `%{foreman_nodejs_sitelib}` macro.

Reply:
We only use it in two places right now so I am less inclined to go that route. This works and is localized to it's use right now. I think we can explore the macro when this path starts to be more widely needed.

Reply:
I'm ok with that. My main concern is that `_scl_root` should already be defined and this is just redefining it. Possibly with a different value.